src/attestedclaim/AttestedClaim.spec.ts

```diff
@@ -12,26 +14,41 @@ function buildAttestedClaim(
   legitimations: AttestedClaim[]
 ): AttestedClaim {
   // create claim
-  const claim = {
-    cType: ctype,
-    contents,
-    owner: claimer.address,
-  } as IClaim
+
+  const contentsCopy = contents
```

Comment:
`contentsCopy` is now pointing to the same object as `contents`.
```javascript
const obj1 = {a: 1, b: 2}
const obj2 = obj1
obj2.a = 3 // now obj1 is {a: 3, b: 2}
```

Do you want to copy the object by value instead? If so, use `{...contents}`. 
Otherwise, you can remove this line and use `contents` directly.